File: backend/src/Domain/Entites/Chore.cs
using Domain.Enums;

namespace Domain.Entites;

public class Chore
{
  public int Id { get; set; }
  public string Title { get; set; } = String.Empty;
  public string? Description { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime? CompletedAt { get; set; }
  public ChoreStatus Status { get; set; } = ChoreStatus.Pending;
}

File: frontend/types/chore.ts
export type ChoreStatus = 'pendente' | 'emProgresso' | 'concluida';

export interface ChoreFormValues {
  title: string;
  description: string;
  status: ChoreStatus;
  completionDate: string | null;
}

export interface ChoreReadDto {
  id: number;
  title: string;
  description: string;
  status: ChoreStatus;
  completionDate: string | null;
  createdAt: string;
}
